sql/pg_duckdb--0.0.1.sql

```diff
@@ -139,12 +139,47 @@ CREATE TABLE extensions (
     enabled BOOL DEFAULT TRUE
 );
 
+CREATE TABLE tables (
```

Comment:
I am a little confused why we need this, I thought a Relation already contained a pointer to the table AM struct, which we can use to check if it's a duckdb temp table?